.changeset/commit.js

```diff
@@ -0,0 +1,24 @@
+const { execSync } = require('node:child_process')
```

Comment:
AH thats annoying... Does changesets still offer enough value if we have to do things like this?

Reply:
Not sure -- it seems the amount of scripts is now similar to our previous approach. It is a bit less hacky which I like. But I'll keep looking further to an alternative. Maybe NX, or back to the lerna setup.